lib/order.rb

```diff
@@ -1,3 +1,8 @@
+require 'csv'
+require 'pry'
+require 'awesome_print'
+
+FILE_NAME = "../support/orders.csv"
 module Grocery
```

Comment:
You should omit the `../` from this path so that `rake` can run. If you need to files individually, you should do so from the project root. For example:

```
$ ls
README.md  Rakefile  feedback.md  lib  seeds.rb  specs  support
$ ruby lib/order.rb
[ ... program output ... ]
```